### PR TITLE
Hides empty age field where value is null and fixes age patient entry

### PR DESCRIPTION
**Pull request by DevChima**

### CODE REVIEW

**Comment by DevChima** [comment]
In hindsight, I'm thinking this `{{ patiententry.age|default_if_none:"-" }}` could simply be `{{ patiententry.age }}` since it would never show a default value anyway if it's null because of the IF condition.

File: cspatients/templates/cspatients/patient.html [hunk 1]
```diff
@@ -21,10 +21,12 @@ <h4 class="heading heading__subtitle">Personal Details</h4>
             <h5 class="patient-list__label">Surname</h5>
             <h4 class="patient-list__entry">{{ patiententry.surname }}</h4>
           </li>
+          {% if patiententry.age is not None %}
```
Comment (nathanbegbie) [inline cspatients/templates/cspatients/patient.html:24]: @DevChima these changes seem contrary to each other.
If `patiententry.age` is `None` then the `default_if_none` filter is not necessary, OR, if it's `None`, we _do_ want to show the AGE, but we want to show that it's blank.

I might be misunderstanding here, please let me know . . .

**Review by nathanbegbie — CHANGES_REQUESTED** [review]

File: cspatients/templates/cspatients/patient.html (hunk 1 above)
Comment (nathanbegbie) [reply cspatients/templates/cspatients/patient.html:24]: Ah, I see you've answered this in the main thread 😅

**Review by erikh360 — APPROVED** [review]
Looks good!

**Pull request merged**